packages/bitcore-wallet-service/src/lib/expressapp.ts

```diff
@@ -1621,6 +1621,116 @@ export class ExpressApp {
       });
     });
 
+    router.get('/v3/device', (req, res) => {
+      let server;
+      try {
+        server = getServer(req, res);
+      } catch (ex) {
+        return returnError(ex, res, req);
+      }
+      server.getAllLogDevice((err, listDevice) => {
```

Comment:
Do we need the api to get all? If there're too many records should we use it without any kind of filter?

Reply:
I will add TODO in case to consider in the future can change to reasonable